xlators/protocol/server/src/server-rpc-fops_v2.c

```diff
@@ -867,6 +868,14 @@ server4_setxattr_cbk(call_frame_t *frame, void *cookie, xlator_t *this,
         goto out;
     }
 
+    if (dict_get_sizen(state->dict, GF_NAMESPACE_KEY)) {
+        /* This inode onwards we will set namespace */
+        gf_msg(THIS->name, GF_LOG_INFO, 0, PS_MSG_SETXATTR_INFO,
```

Comment:
This should be debug level at most.

Reply:
This did help in knowing how many namespace requests came on the directory. But yes, will move it to DEBUG level